src/platform/sdl/gl-common.c

```diff
@@ -46,4 +46,7 @@ void mSDLGLCommonInit(struct mSDLRenderer* renderer) {
 	SDL_SetVideoMode(renderer->viewportWidth, renderer->viewportHeight, 32, SDL_OPENGL | (SDL_FULLSCREEN * renderer->fullscreen));
 #endif
 #endif
+	if (renderer->lockIntegerScaling) {
+		SDL_SetWindowMinimumSize(renderer->window, renderer->width, renderer->height);
+	}
```

Comment:
I'm not sure about this fix. But I can't think of another way, and I'd rather not let the screen go blank by resizing it too small.